src/content/self-hosted/administration/configuration.mdx

```diff
@@ -47,6 +47,26 @@ subdomain: "example.com"
 
 After installation, we recommend that you create a DNS entry for `*.$SUBDOMAIN`, pointing to the public address of your load balancer.
 
+### publicOverride
+
+Use this property to override where Okteto is available. This option replaces `okteto.$SUBDOMAIN` with you [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of choice.
```

Comment:
@jpf-okteto @pchico83 we also need to create a certificate for this FQDN right? can we include instructions on that?